src/constants/card-app.ts

```diff
@@ -8,27 +8,70 @@ interface CardMetaInfo {
 
 export const CARD_META_INFO: Record<keyof CardInfo, CardMetaInfo> = {
   cardNumbers: {
-    query: '결제할 카드 번호를 입력해 주세요.',
+    query: '결제할 카드 번호를 입력해 주세요',
     label: '카드 번호',
     caption: '본인 명의의 카드만 결제 가능합니다.',
   },
   expirationDate: {
-    query: '카드 유효기간을 입력해 주세요.',
+    query: '카드 유효기간을 입력해 주세요',
     label: '유효기간',
     caption: '월/년도(MMYY)를 순서대로 입력해 주세요.',
   },
   cardOwnerName: {
-    query: '카드 소유자 이름을 입력해 주세요.',
+    query: '카드 소유자 이름을 입력해 주세요',
     label: '소유자 이름',
     caption: '',
   },
+  cardCompany: {
+    query: '카드사를 선택해 주세요',
+    label: '',
+    caption: '현재 국내 카드사만 가능합니다.',
+  },
+  cardCVCNumber: {
+    query: 'CVC 번호를 입력해 주세요',
+    label: 'CVC',
+    caption: '',
+  },
+  cardPassword: {
+    query: '비밀번호를 입력해 주세요',
+    label: '앞의 2자리를 입력해 주세요.',
+    caption: '비밀번호 앞 2자리',
+  },
+};
+
+export const CARD_COMPANIES = [
+  'BC카드',
+  '신한카드',
+  '카카오뱅크',
+  '현대카드',
+  '우리카드',
+  '롯데카드',
+  '하나카드',
+  '국민카드',
+];
+
+interface CardColor {
+  [key: string]: string;
+}
+
+export const CARD_COLOR: CardColor = {
```

Comment:
카드 회사 정보를 타입으로 관리할 수 있다면, 이 부분도 바뀔 수 있겠죠!
```suggestion
export const CARD_COLOR: Record<CardCompany, string> = {
```

Reply:
step1에서 사용했던 Record 타입을 여기서도 적용할 수 있겠네요!!